backend/case_designer/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from books.models import Book
+
+DISPLAY_MODE_CHOICES = [
+    ('Spine Out', 'Spine Out'),
+    ('Cover Out', 'Cover Out')
+]
+
+class Bookcase(models.Model):
+    name=models.CharField(max_length=50)
+    last_edit_date=models.DateTimeField(auto_now=True)
+    width=models.FloatField()
+    # To be completed when users are implemented
+    #creator=models.ForeignKey('auth.User', on_delete=models.CASCADE)
```

Comment:
I think we can integrate Users. Also we use a custom class for users at `authapp.User`